subprojects/docs/src/docs/release/notes.md

```diff
@@ -24,6 +24,15 @@ Think of every feature section as a mini blog post.
   NOTE: Totally fine to just link to an example that show the feature.
 -->
 
+### Deprecation warning improvements
+
+In this release, deprecation warnings are no longer displayed in the build by default. Instead, all deprecation warnings in the build will be collected and a single "summary" will be rendered at the end of the build:
+
+    There're X deprecation warnings, which may break the build in Gradle 5.0. Please run with --warnings=all to see them.
+
+As said by this warning, you can run the build with command line option `--warnings=all` or property `org.gradle.warnings=all` to have all warnings displayed as before.
```

Comment:
:o: I am not quite sure what we mean by "as before". I guess you mean in earlier versions of Gradle?

Reply:
Done.